### ISSUE TITLE
Update light theme focus-glow to correct focus color

### ISSUE THREAD

**Issue by bobbyomari** [dequelabs/cauldron#918]
`--focus-glow` is using an old hexcode and should be updated to `rgba(181, 26, 209, .75);` which is `--focus-light` at 75% opacity.
